pkg/cmd/importcmd/import.go

```diff
@@ -382,12 +382,22 @@ func (options *ImportOptions) Run() error {
 	}
 	options.AppName = naming.ToValidName(strings.ToLower(options.AppName))
 
+	if !options.DisableDraft {
+		// Disable draft if jenkins-x.yaml is already present
+		jenkinsxFile := filepath.Join(options.Dir, "jenkins-x.yml")
+		_, err = os.Stat(jenkinsxFile)
```

Comment:
For what it's worth there is a helper `FileExists` in the util package for this.